scripts/check

```diff
@@ -9,6 +9,6 @@ export SOURCE_FILES="httpx tests"
 set -x
 
 ./scripts/sync-version
-${PREFIX}black --check --diff $SOURCE_FILES
+${PREFIX}ruff format $SOURCE_FILES --check 
```

Comment:
```suggestion
${PREFIX}ruff format $SOURCE_FILES --diff
```

Reply:
Does ruff format support --diff option?

```
ruff format test.py --check --diff
error: unexpected argument '--diff' found

Usage: ruff format <FILES|--check|--config <CONFIG>|--respect-gitignore|--no-respect-gitignore|--force-exclude|--no-force-exclude|--line-length <LINE_LENGTH>|--isolated|--stdin-filename <STDIN_FILENAME>|--preview|--no-preview>

```

Reply:
> Does ruff format support --diff option?

Yes.
```
>ruff --version
ruff 0.1.2

>ruff help format
Run the Ruff formatter on the given files or directories

Usage: ruff format [OPTIONS] [FILES]...

Arguments:
  [FILES]...  List of files or directories to format

Options:
      --check
          Avoid writing any formatted files back; instead, exit with a non-zero status code if any files would have been modified, and zero otherwise
      --diff
          Avoid writing any formatted files back; instead, exit with a non-zero status code and the difference between the current file and how the formatted file would look like

...
```